File: PYTHON PROGRAMMING/Test69.py
# 69. PRIME NUMBERS FROM 0 TO N
from math import *
def Prime(num):
    flag = True
    for i in range(2, floor(sqrt(num + 1))):
        if num % i == 0: flag = False
    return flag
def PrimeSeries(limit):
    for i in range(2, limit + 1):
        if Prime(i):
            print(i, end=" ")
    print()
PrimeSeries(100)
PrimeSeries(200)

File: JAVASCRIPT PROGRAMMING/Test69.js
"use strict";
// 69. PERFECT SQUARE NUMBER
const perfectSquare = (num) => {
    let sqrtnum = Math.floor(Math.sqrt(num));
    let flag = (sqrtnum * sqrtnum === num);
    let res = (flag) ? `${num} Is A Perfect Square Number.` :
        `${num} Is Not A Perfect Number.`;
    console.log(res);
};
perfectSquare(36);
perfectSquare(15);